locals.tf

```diff
@@ -0,0 +1,3 @@
+locals {
```

Comment:
If you are using locals then try to refrence values for all the variables as well.